extension/content_script.js

```diff
@@ -149,49 +143,41 @@ function calculateTrackerLife() {
 	let distributable_time = desired_time_s - base_time_s; // Time left to distribute to sentences
 	let word_ratio = sentence_words/total_words_remaining;
 	let linger_time_ms = distributable_time*(word_ratio)*1000 + speed_bias_ms; // convert from s to ms
+
 	return (linger_time_ms); 
 	// TODO: Use Moment.js
 }
 
 /*
-Move one sentence up.
+Move one sentence in the given direction.
 Return:
 - Boolean: True iff tracker successfully moved. False if there is no more element to move to.
 */
-function moveUpOne() { // Sets start and end
+function moveOne(dir) { // Sets start and end
 	// Let scrolling finish before any movement.
 	if (isScrolling) {
 		return;
 	}
-	let hasMoved = tracker.movePrevious();
-	if (!hasMoved) {
-		return false;
-	}
-	highlight(tracker);
-	scrollUp();
-	return true;
-}
 
-/*
-Move one sentence down.
-Return:
-- Boolean: True iff tracker successfully moved. False if there is no more element to move to.
-*/
-function moveDownOne() { // Sets start and end
-	// Let scrolling finish before any movement.
-	if (isScrolling) {
-		return;
+	let hasMoved = false;
+
+	if (dir == direction.BACKWARD) {
+		hasMoved = tracker.movePrevious();
+	} else if (dir == direction.FORWARD) {
+		hasMoved = tracker.moveNext();
 	}
-	let hasMoved = tracker.moveNext();
 	if (!hasMoved) {
 		return false;
 	}
-
-	let sentenceId = tracker.getSentenceId();
-	let sentencePtr = doc.getSentence(sentenceId);
-	display.updateTimer(doc.getContainers(), sentencePtr.containerId);
+	display.updateTimer(tracker.getSentenceId());
```

Comment:
Decided to combine into 1 fcn because a lot of bookkeeping of moveup / movedown is shared, including the updating of the display timer.